migrations/1625095491619-CreateBlockedAddressesTable.ts

```diff
@@ -0,0 +1,23 @@
+import { MigrationInterface, QueryRunner, Table, TableIndex } from 'typeorm';
+
+const blockedAddressesTable = new Table({
+    name: 'blocked_addresses',
+    columns: [
+        { name: 'address', type: 'varchar', isPrimary: true },
```

Comment:
might be worth forcing to lowercase to avoid duplicate addrs

Reply:
nice - good thought, will add a check constraint